app/main.py

```
@@ -0,0 +1,102 @@
+
+    @app.get("/ready", tags=["ops"])  # Deeper readiness
+    def ready() -> dict[str, str]:
+        """Return readiness signal based on configuration checks.
+
+        Extend this to validate external dependencies (DB, Redis, etc.).
+        """
+        try:
+            # Minimal readiness: application settings can be loaded.
+            _ = get_application_settings()
+            readiness_gauge.set(1)
+            return {"status": "ready"}
```

Ruff (0.13.1)

Consider moving this statement to an `else` block

(TRY300)